### ISSUE TITLE
Enable inferencing for various model frameworks/outputs

### ISSUE THREAD

**Issue by nrweir** [CosmiQ/cw-nets#56]
Will need to look into what the outputs are like for various different models and see how I should store them, as well as how inference is run for:

- [ ] PyTorch
- [ ] tf/keras
- [ ] Darknet

Ideally I'd like to be able to enable sliding windows to cover images for things like segmentation masks.

**Comment by nrweir** [comment]
Issue moved to [CosmiQ/solaris #23](https://github.com/CosmiQ/solaris/issues/23) via [**ZenHub**](https://www.zenhub.com/)
